### PR TITLE
Enable reauth for Realms.

### PR DESCRIPTION
**Pull request by rickybrent**
Thanks for this great mod! 

It works great with Realms servers currently, but only if you never click "realms" while your login is expired -- as soon as it fails once, Realms never checks again. This should fix that and add the button to the "Invalid session" realms page, too. :)

### CODE REVIEW

File: src/main/java/me/axieum/mcmod/authme/mixin/RealmsMainScreenMixin.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(RealmsMainScreen.class)
+public interface RealmsMainScreenMixin
+{
+    @Accessor(value="checkedClientCompatability")
+    @Mutable
+    static void setCheckedClientCompatability(boolean checkedClientCompatability){
+        // Method body is ignored.
+        throw new UnsupportedOperationException();
```
Comment (Axieum) [inline src/main/java/me/axieum/mcmod/authme/mixin/RealmsMainScreenMixin.java:22]: I think we can simplify both of these down to be like [`SetSessionMixin`](https://github.com/Axieum/AuthMe/blob/master/src/main/java/me/axieum/mcmod/authme/mixin/SetSessionMixin.java)

File: src/main/java/me/axieum/mcmod/authme/util/SessionUtil.java [hunk 2]
```diff
@@ -158,6 +160,9 @@ private static void setSession(Session session) throws IllegalAccessException
     {
         // NB: Minecraft#session is a final property - use mixin accessor
         ((SetSessionMixin) MinecraftClient.getInstance()).setSession(session);
+        // Necessary for Realms to re-check for a valid session.
```
Comment (Axieum) [inline src/main/java/me/axieum/mcmod/authme/util/SessionUtil.java:163]: Being really picky here, but could we add a space above and remove the period/full-stop 😛

File: src/main/java/me/axieum/mcmod/authme/mixin/RealmsMainScreenMixin.java (hunk 1 above)
Comment (rickybrent) [reply src/main/java/me/axieum/mcmod/authme/mixin/RealmsMainScreenMixin.java]: Done! :)

File: src/main/java/me/axieum/mcmod/authme/util/SessionUtil.java (hunk 2 above)
Comment (rickybrent) [reply src/main/java/me/axieum/mcmod/authme/util/SessionUtil.java]: Nothing wrong with being picky about code style -- done! :)

**Pull request merged**